ebs/ebs.go

```diff
@@ -580,7 +582,9 @@ func (d *Driver) CreateVolume(req Request) error {
 	if err != nil {
 		return err
 	}
-	d.UpdateTags(volumeID, newTags)
+	if err := d.UpdateTags(volumeID, newTags); err != nil {
```

Comment:
If I remember well the log statement in `UpdateTags()` logs the error if any and says that it will still continue. If we are not doing that anymore (by making this a strict requirement) then it would be worth changing that log statement too.